app/models/manageiq/providers/embedded_ansible/automation_manager/playbook_runner.rb

```diff
@@ -0,0 +1,140 @@
+class ManageIQ::Providers::EmbeddedAnsible::AutomationManager::PlaybookRunner < ::Job
+  # options are job table columns, including options column which is the playbook context info
+  def self.create_job(options)
+    super(name, options)
+  end
+
+  def start
+    time = Time.zone.now
+    update_attributes(:started_on => time)
+    miq_task.update_attributes(:started_on => time)
+    if options[:inventory]
+      queue_signal(:create_job_template)
+    else
+      queue_signal(:create_inventory)
+    end
+  end
+
+  def create_inventory
+    set_status('creating inventory')
+    tower = playbook.manager
+    hosts = options[:hosts] || options.fetch_path(:extra_vars, :hosts)
+    options[:inventory] =
+      if hosts == 'localhost' || hosts.nil?
+        tower.provider.default_inventory
+      else
+        inventory_name = "#{playbook.name}_#{Time.zone.now.to_i}"
+        ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Inventory.raw_create_inventory(tower, inventory_name, hosts).id
+      end
+    save!
+    queue_signal(:create_job_template)
+  rescue => err
+    _log.log_backtrace(err)
+    queue_signal(:post_ansible_run, err.message, 'error')
+  end
+
+  def create_job_template
+    set_status('creating job template')
+    raw_job_template = playbook.raw_create_job_template(options)
+    options[:job_template_ref] = raw_job_template.id
+    save!
+
+    queue_signal(:launch_ansible_tower_job)
+  rescue => err
+    _log.log_backtrace(err)
+    queue_signal(:post_ansible_run, err.message, 'error')
+  end
+
+  def poll_ansible_tower_job_status(interval)
```

Comment:
@bzwei Minor, but can we swap the order the methods are listed in the file to match the "happy path" the methods would take.  This would mean swapping the order that `launch_ansible_tower_job` and `poll_ansible_tower_job_status` appear in this file.

Also, fix the rubocop warning on line 57 with the extra space.